nixos.nix

```diff
@@ -332,7 +341,10 @@ in
           value = {
             device = concatPaths [ persistentStoragePath directory ];
             noCheck = true;
-            options = [ "bind" ];
+            options = [ "bind" ] ++ (if cfg.${persistentStoragePath}.hideMounts then
```

Comment:
This can be more concisely written with `optional` or `optionals` -- e.g. `++ optional cfg.${persistentStoragePath}.hideMounts "x-gvfs-hide";` or `++ optionals .... [ "x-gvfs-hide" ];`